deployment/kuadrant-openshift/kustomize/prometheus/grafana-rbac.yaml

```
@@ -0,0 +1,46 @@
+- kind: ServiceAccount
+  name: grafana-sa
+  namespace: maas-observability
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: grafana-sa-token
+  namespace: maas-observability
+  annotations:
+    kubernetes.io/service-account.name: grafana-sa
+type: kubernetes.io/service-account-token
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)